server/routes/v1/version.ts

```diff
@@ -99,3 +99,40 @@ export const resetVersionApprovals = [
     return res.json(version)
   },
 ]
+
+export const updateLastViewed = [
+  ensureUserRole('user'),
+  async (req: Request, res: Response) => {
+    const { id, role } = req.params
+    const { user } = req
+    if (!user) {
+      throw Forbidden({ code: 'user_unauthorised' }, 'Not user details found in request.')
+    }
+    const version = await findVersionById(req.user!, id, { populate: true })
+    if (!version) {
+      throw BadReq({ code: 'version_not_found' }, 'Unabled to find requested version')
+    }
+    if (user?.id !== version?.metadata.contacts[role]) {
```

Comment:
`user` should be guaranteed at this point as we've guarded against `if (!user)` above. Should be able to remove the null access `?`